File: URI/1 - INICIANTE/C/1188 - AreaInferior.c
#include <stdio.h>

int main(){
	char op;
	double matriz[12][12], soma;
	int linhaCont, colunaCont, posicaoInicial = 6, posicaoFinal = 6,cont = 0;
	scanf(" %c", &op);
	
	for(linhaCont = 0; linhaCont < 12; linhaCont++){
		for(colunaCont = 0; colunaCont < 12; colunaCont++){
			scanf(" %lf", &matriz[linhaCont][colunaCont]);
		}
	}
	for(linhaCont = 7; linhaCont < 12; linhaCont++){
		posicaoInicial -= 1;
		posicaoFinal += 1;
		for(colunaCont = posicaoInicial; colunaCont < posicaoFinal; colunaCont++){
			soma += matriz[linhaCont][colunaCont];
			cont += 1;
		}
	}
	if(op == 'S'){
		printf("%.1lf\n", soma);
	}else{
		printf("%.1lf\n", soma/cont);
	}
	
	return 0;
}

File: URI/1 - INICIANTE/Python/1188 - AreaInferior.py
op = input()
matriz = []
soma = 0.0
posicaoInicial = 6
posicaoFim = 6
total = 0
for x in range(0, 12):
    linha = []
    for y in range(0, 12):
        linha.append(float(input()))
    matriz.append(linha)
for linhaCont in range(7, 12):
    posicaoInicial -= 1
    posicaoFim += 1
    for colunaCont in range(posicaoInicial, posicaoFim):
        soma += matriz[linhaCont][colunaCont]
        total += 1
if op == 'S':
    print('{:.1f}'.format(soma))
else:
    print('{:.1f}'.format(soma/total))